sos/policies/ubuntu.py

```diff
@@ -8,6 +8,8 @@ class UbuntuPolicy(DebianPolicy):
     distro = "Ubuntu"
     vendor = "Canonical"
     vendor_url = "https://www.ubuntu.com/"
+    PATH = "/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games" \
+           + ":/usr/local/sbin:/usr/local/bin:/snap/bin"
```

Comment:
@TurboTurtle @bmr-cymru @pmoravec @BryanQuigley Just thinking out loud after a discussion I had with Bryan. Would it make sense to you in a near future to stop hardcoding PATH for policies and rely on system PATH environment variable instead ? and fallback to this hardcoded one if no system PATH env variable found or empty ?

Reply:
In the same time it can be misleading depending how bad the system is impacted by problem 'X'. Maybe the hardcoded PATH is a safety net to make sure the PATH will be reliable in case the system is too broken or that we can't rely on system PATH env ?

Reply:
Right, that's the problem: we introduced `PATH` in policies for two reasons. To avoid the need to maintain absolute paths in plugin code and so that whatever messed up environment the user might have when running sos doesn't affect the execution of commands by the tool.

This was at the time of the "UsrMove" change in Red Hat and Fedora: prior to this we used absolute paths in many places in sos. Having a stable `PATH` defined by policies allowed us to simplify plugin and increase code sharing by just using the binary name and to ensure that we would be able to find executables even if the user's PATH was incorrectly set.

I considered adding something to obtain the path by parsing/interpreting `/etc/profile` for Red Hat distributions but that gets quite complex (and there's no guarantee that the file hasn't been modified/broken, so we would still need some kind of heuristic to validate the resulting path list).

Reply:
Gotcha, thanks.